lib/modules/about.js

```diff
@@ -10,88 +10,77 @@ module.sort = -10;
 module.alwaysEnabled = true;
 module.description = aboutTemplate();
 
-module.options = {};
```

Comment:
Moving these into a single object will be taken care of by https://github.com/erikdesjardins/Reddit-Enhancement-Suite/commit/348913b062a16ceeff043b62cae5c5e8c751875a#diff-9a6bec128b40a3ba6580d95a3a4f54b0 (which also includes other changes to those options in some places) so let's leave this out for now to avoid causing conflicts. It will also conflict with i18n'ing the option descriptions